packit/status.py

```diff
@@ -65,11 +67,12 @@ def get_downstream_prs(self, number_of_prs: int = 5) -> List[Tuple[int, str, str
             table = [(pr.id, pr.title, pr.url) for pr in pr_list]
         return table
 
-    def get_dg_versions(self) -> None:
+    def get_dg_versions(self) -> Dict:
         """
         Get versions from all branches in Dist-git
-        :return: None
+        :return: Dict {"branch": "version"}
```

Comment:
not too happy about this but I guess that we don't need to invent a new type here